src/main/java/org/osiam/auth/login/internal/InternalAuthenticationProvider.java

```diff
@@ -61,16 +62,19 @@
 
     private final SCIMUserProvisioning userProvisioning;
     private final ShaPasswordEncoder passwordEncoder;
```

Comment:
I would suggest that you rename the old `passwordEncoder` to `shaPasswordEncoder`, to make obvious what's happening